README.md

```diff
@@ -1,2 +1,15 @@
 # husky-musher
 A redirecting service for the Husky Coronavirus Testing study
+
+## Running locally
+Run the Flask app with:
+```sh
+FLASK_APP=app.py run flask run
```

Comment:
Missing a `pipenv` here. `FLASK_APP` is also unnecessary; `app.py` is one of the files `flask run` will look for automatically.

Reply:
Ah yes I wasn't sure if we wanted to be explicit about invoking `pipenv` because we don't always use it in our code examples. I can change this example, though.

Reply:
Addressed in a new commit.

Reply:
Nod. I'm fine with either `pipenv run flask run` or `flask run`, but the original was `run flask run`. :-)

Reply:
Oh I see, haha :running_woman: